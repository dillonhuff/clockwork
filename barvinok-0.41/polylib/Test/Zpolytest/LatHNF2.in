# HNF of this Matrix (not a lattice)
M 1
9 5
0 0 0 1 0
4 0 4 0 -4
12 15 0 0 -3
1 0 -1 0 10
0 0 0  0 1
4 0 4 0 -4
12 15 0 0 -2
1 7 -1 0 10
0 0 0  0 1
D 0
F 2

m1=Alpha`matrix[5,5,{},
{{0, 0, 0, 1,0}, {4, 0, 4, 0,-4},{12, 15, 0, 0,-3},{1, 0, -1, 0,10 },{0, 0, 0,  0, 1}}]
hermiteL[m1]