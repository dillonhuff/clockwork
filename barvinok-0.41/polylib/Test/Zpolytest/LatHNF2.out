9 5
   1   0   0   0   0
   0   4   0   0   0
   0   0   3   0   0
   0   9   8  10   7
   0   0   0   0   1
   0   4   0   0   0
   0   0   3   0   1
   0 -19 -13 -18  14
   0   0   0   0   1
5 5
   0   0   0   1   0
   1   0   1   0  -1
   4   5   0   0  -1
  -4  -4  -1   0   2
   0   0   0   0   1
